index1.html

```diff
@@ -0,0 +1,328 @@
+        <link href="https://fonts.googleapis.com/css?family=Inter:100,200,300,regular,500,600,700,800,900" rel="stylesheet" />
+    
+        <link href="style.css" rel="stylesheet">
+        <link href="css/all.min.css" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+      
+    
+        <title>Hahimi Portfolio</title>
+    </head>
+    
+    <body>
+        <header >
+            <div class="logo">
+                <a href="#">Zuhra</a>
+    
+            </div>
+            <nav class="navbar ">
+
+                <ul class="desktop-nav" id="desktop-nav">
+                   
+                    <li class="nav-link" id="nav-link"><a href="#recent-works ">Portfolio</a></li>
+                    <li class="nav-link"><a href="#about-me ">About</a></li>
+                    <li class="nav-link"><a href="#contact-me">Contact Us</a></li>
+                    <li class="envelope" id="envelope"><a href="#"><i class="fas fa-envelope" ></i></a></li>
+                </ul>  
+                <div class="humberger-menu">
+                    <div class="line1"></div>
+                    <div class="line2"></div>
+                    <div class="line3"></div>
+                  </div>
+                
+            </nav>
+        </header>
+        <main>
+            <section class="landing">
+                <div class="intro ">
+                    <h1>Hey there. <br>I am Zuhra Hashimi</h1>
+                    <h2>I am a Software Developer</h2>
+                    <p>I can help you build a product , feature or website Look
+                         through some of my work and experience! If you like what
+                          you see and have a project you need coded, don’t hestiate 
+                          to contact me.</p>
+                </div>
+                <div class="social-icon">
+                    <ul>
+                        <li><a href="#"><i class="fab fa-github-alt "></i></a></li>
+                        <li><a href="#"><i class="fab fa-linkedin "></i></a></li>
+                        <li><a href="#"><i class="fab fa-twitter-square "></i></a></li>
+                        <li><a href="#"><i class="fab fa-youtube "></i></a></li>
+                        <li><a href="#"><i class="fab fa-medium "></i></a></li>
+                    </ul>
+                </div>
+            </section>
+             <!-- =============
+            Recent Works section
+              =========== -->
+        <section class="recent-works">
+            <div class="first">
+                <h3>My Recent Works</h3>
+                <hr>
+            </div>
+            <div class="recent-work" id="recent-works">
+                <a href="" class="women" ><img src="./image/Img_Placeholder_1.png" alt="Women doing yoga"></a>
```

Comment:
This image is breaking, consider having a look at the source of the image. 
![image](https://user-images.githubusercontent.com/33268927/170816561-9fd121b5-cb97-4c1a-8ea4-8cc85621150e.png)